apps/docs/src/app/platform/component-docs/platform-split-menu-button/platform-split-menu-button-examples/platform-split-menu-button-examples.component.html

```diff
@@ -7,7 +7,7 @@ <h3>Cozy Split Menu Button</h3>
         [buttonLabel]="label"
         [icon]="'world'"
         [type]="buttonType"
-        [title]="'default button with menu'"
+        [title]="'Default action button with list of other available action items'"
         [menuTitle]="'More options'"
         (primaryButtonClick)="onPrimaryButtonClickCozy()"
         i18n
```

Comment:
if this is not an i18n example, can you remove the marker from here or add an id like you did for the other example?